.vib/common/goss/templates/check-apache-libphp.yaml

```
@@ -0,0 +1,15 @@
+command:
+  # Check explicitly enabled and disabled modules
+  check-loaded-modules:
```

Rename it in order not to collide with other defined tests

```suggestion
  check-apache-libphp-loaded-modules:
```